onnx/test/shape_inference_test.py

```diff
@@ -1100,6 +1121,60 @@ def test_dropout(self):  # type: () -> None
     def test_LRN(self):  # type: () -> None
         self._identity_prop('LRN', alpha=0.5, beta=0.5, size=1)
 
+    def test_shrink(self):  # type: () -> None
+        graph = self._make_graph(
+            [('x', TensorProto.FLOAT, (3, 4, 5))],
+            [make_node('Shrink', 'x', 'y')],
+            [])
+        self._assert_inferred(graph, [make_tensor_value_info('y', TensorProto.FLOAT, (3, 4, 5))])
+
+    def test_TfIdfVectorizer_1D(self):  # type: () -> None
+        graph = self._make_graph(
+            [('x', TensorProto.FLOAT, [12])],
+            [make_node('TfIdfVectorizer',
+                       'x',
+                       'y',
+                       mode='TF',
+                       min_gram_length=2,
+                       max_gram_length=2,
+                       max_skip_count=0,
+                       ngram_counts=[0, 4],
+                       ngram_indexes=[0, 1, 2, 3, 4, 5, 6],
+                       pool_int64s=[2, 3, 5, 4, 5, 6, 7, 8, 6, 7])],
+            [])
+        self._assert_inferred(graph, [make_tensor_value_info('y', TensorProto.FLOAT, [7])])
+
+    def test_TfIdfVectorizer_2D(self):  # type: () -> None
+        graph = self._make_graph(
+            [('x', TensorProto.FLOAT, [15, 12])],
+            [make_node(
+                'TfIdfVectorizer',
+                'x',
+                'y',
+                mode='TF',
+                min_gram_length=2,
+                max_gram_length=2,
+                max_skip_count=0,
+                ngram_counts=[0, 4],
+                ngram_indexes=[0, 1, 2, 3, 4, 5, 6],
+                pool_int64s=[2, 3, 5, 4, 5, 6, 7, 8, 6, 7])],
+            [])
+        self._assert_inferred(graph, [make_tensor_value_info('y', TensorProto.FLOAT, [15, 7])])
```

Comment:
```suggestion
        self._assert_inferred(graph, [make_tensor_value_info('y', TensorProto.FLOAT, (15, 7))])
```
If previously we use tuples for representing shapes, we can still use them for consistency.